backend.go

```diff
@@ -21,6 +21,26 @@ type Backend interface {
 	AnonymousLogin(state *ConnectionState) (Session, error)
 }
 
+type Feature uint32
+
+const (
+	// SMTPUTF8 (RFC 6531) extension.
+	FeatureSMTPUTF8 Feature = 2 << iota
+	// BINARYMIME (RFC 3030) extension. CHUNKING extension is always supported.
+	FeatureBINARYMIME
```

Comment:
The rest of the code seems to use the Go capitalization: "BinaryMIME"

(Same for "RequireTLS" I suppose)